### ISSUE TITLE
motion detector not working propperly

### ISSUE THREAD

**Issue by Astado** [jheling/freeathome#171]
Hi all,
first thanks for all the hard work on this project!!! I really would be devastated without it. It made my home so much smarter. :)

Now my issue.
I would very much like to include my motion detectors into my alarm system. At the moment they are displayed in home assistant but always as "on". It doesn't matter if a motion is detected or not.
Is there a way to fix this?

**Comment by SimonKbg** [comment]
Hi,
I created an input_boolean helper in Home Assistant which was exposed to free@home using the emulated Hue Bridge. In free@home this helper was recognized as a switch. Then I connected the motionsensor with this „switch“ which was then triggered an recognized in HA.
Hope that helps.

Off topic problem for me: the emulated hue bridge stopped working after updating SysAP to 3.2.3…
Best,
Simon

**Comment by jheling** [comment]
The integration can only intercept the messages of a motion detector if it is connected to another device.  
The input boolean helper SimonKgb describes is a good option.

**Comment by Astado** [comment]
Hi,

sorry for the late answer.
I tried out the embedded hue bridge in home assistant and it works perfect. 
Thanks to you guys!

For those who need to do the same.
In the configuration.yaml I added the followingg lines:
emulated_hue:
  host_ip: your_ip
  listen_port: 8300 
  expose_by_default: true
  exposed_domains:
    - input_boolean
  entities:
    input_boolean.motion_sensor_1:
      name: "motion sensor 1"
    input_boolean.motion_sensor_2:
      name: "motion sensor 2"

In f@h I then got a notification that a bridge was found. I connected with the bridge.
The devices can be found in f@h and added to a room. I then connected the motion sensor to the device.
